### PR TITLE
dojson: language support

### PR DESCRIPTION
**Pull request by kaplun**
* Normalizes languages using ISO 639-1.

Signed-off-by: Samuele Kaplun <samuele.kaplun@cern.ch>

### CODE REVIEW

File: inspirehep/modules/records/jsonschemas/records/hep.json [hunk 1]
```diff
@@ -514,7 +514,9 @@
         "languages": {
             "items": {
                 "title": "Language",
-                "type": "string"
+                "type": "string",
+                "pattern": "\\w{3}",
+                "format": "ISO 639-2"
```
Comment (jacquerie) [inline inspirehep/modules/records/jsonschemas/records/hep.json]: Leftover : (

Comment (kaplun) [inline inspirehep/modules/records/jsonschemas/records/hep.json]: Thanks!

File: tests/unit/dojson/test_dojson_hep.py [hunk 2]
```diff
@@ -116,6 +116,77 @@ def test_external_system_numbers(marcxml_to_json, json_to_marc):
...
+    snippet = (
+        '<datafield tag="041" ind1=" " ind2=" ">'
+        '  <subfield code="a">Italian</subfield>'
+        '</datafield>'
+    )
+
+    expected = ["it"]
+    result = clean_record(hep.do(create_record(snippet)))
+
+    assert expected == result['languages']
```
Comment (jacquerie) [inline tests/unit/dojson/test_dojson_hep.py]: I just had an epiphany: it's easy to write tests that ensure that the rule satisfies the schema! You just need to fetch the schema, drill down to the right point and then call `validate` from `jsonschema` on that subschema. Nothing to fix here, I just wanted to write it down so I don't forget it.

Comment (kaplun) [inline tests/unit/dojson/test_dojson_hep.py]: But in these tests we are additionally testing that the content is correctly propagated (i.e. rule might produce valid schema, but maybe loose data during the translation).

**Pull request merged**